lib/generators/service/service_generator.rb

```diff
@@ -22,18 +23,6 @@ def create_base_dir
     directory 'base', 'app/services'
   end
 
-  def add_controller_helper_to_application_controller
```

Comment:
i think we should keep this method instead of using `ActionController` it's more clear for the user and he can change `command()` method name or modify `ServiceControllerHelper` as he want :thinking:

Reply:
I was planning to add this to the gem configuration initializer. What I'm working on so far looks like this:

```ruby
# config/initializers/command_service_object.rb

CommandServiceObject.configure do |config|
  # By setting the append_controller_helper to false you will need to manually include
  # the CommandServiceObject::ServiceControllerHelper module to your base controller.
  # config.append_controller_helper = true
  #
  # The default command method name is :command
  # but you can easily rename it to avoid collisions.
  # config.command_method_name = :command
  #
  # Skip the generation of commands by the rails g service command
  # config.skip_command = false
  #
  # Skip the generation of minitest/rspec by the rails g service command
  # config.skip_test = false
  #
  # Skip the generation of errors by the rails g service command
  # config.skip_error = true
end
```